loudness/readme.txt

```diff
@@ -1,7 +1,7 @@
 === Loudness ===
 Contributors: Automattic
-Requires at least: 5.8
```

Comment:
Just checking this change is desired?

Reply:
I didn't intend to make that change in this PR but the change was intentional and desired.